config.js

```diff
@@ -245,6 +242,8 @@ var config = {
 
     // UI
     //
+    // When prejoin page is enabled having this flag 'true' will bypass the pre join page.
+    bypassPrejoin: false,
```

Comment:
Given that both options are whitelisted, I don't see why we need 2. Having just the latter is enough, so thhe bypass is essentially overriding its value to false.